React/Fabric/Mounting/ComponentViews/TextInput/RCTTextInputComponentView.mm

```diff
@@ -185,6 +185,13 @@ - (void)updateProps:(Props::Shared const &)props oldProps:(Props::Shared const &
     }
   }
 
+  if (newTextInputProps.traits.smartInsertDelete != oldTextInputProps.traits.smartInsertDelete) {
```

Comment:
Can you add a comment on what this is doing specifically

Reply:
This method is called when updating component's props, so for each prop we compare if the new value is different from the old one. If it is different, we update the native view with the new value.

`RCTUITextSmartInsertDeleteTypeFromOptionalBool` is used to convert the boolean value coming from JS side to the appropriate `UITextSmartInsertDeleteType` value, that is required for this `smartInsertDeleteType` attribute.

Reply:
I meant in the code :D

Reply:
Oh ok, sure 😂

Reply:
Done.